content/docs/forwarding-refs.md

```diff
@@ -1,76 +1,76 @@
 ---
 id: forwarding-refs
-title: Forwarding Refs
+title: Reenvío de Refs
 permalink: docs/forwarding-refs.html
 ---
 
-Ref forwarding is a technique for automatically passing a [ref](/docs/refs-and-the-dom.html) through a component to one of its children. This is typically not necessary for most components in the application. However, it can be useful for some kinds of components, especially in reusable component libraries. The most common scenarios are described below.
+El reenvío de Refs es una técnica para pasar automáticamente una [ref](/docs/refs-and-the-dom.html) a través de un componente a uno de sus hijos. Esto normalmente no es necesario para la mayoría de los componentes en una aplicación. Sin embargo, puede ser util para ciertos tipos de componentes, especialmente en bibliotecas de componentes reutilizables. Los escenarios mas comunes son descritos a continuación.
```

Comment:
@elyalvarado Nice work!
A couple accents missing here: `puede ser útil...` and in `Los escenarios más...` :)

Reply:
thanks, fixed that and other typos. I finally installed a spell checking plugin for my IDE.